rfc.tex

```diff
@@ -1,16 +1,6 @@
 \section{Decision Making Process}
-
-During the research and development phase of the project, DM used a formal process for approving all changes to the DM system.
```

Comment:
👎 getting rid of historical background but you are right it's better to say what we are doing than what we did in the past.